.github/workflows/benchmark_master.yml

```diff
@@ -155,4 +155,86 @@ jobs:
         run: |
           nvm install 20
           npm install -g aws-cdk
-          /bin/bash ./.github/workflow_scripts/run_benchmark.sh multimodal master_medium none g4_12x ${{ env.AG_BRANCH_NAME }} ${{ github.sha }}
+          /bin/bash ./.github/workflow_scripts/run_benchmark.sh multimodal master_medium automm-text g4_12x ${{ env.AG_BRANCH_NAME }} ${{ github.sha }}
```

Comment:
Do we only run on multi-gpu?

Reply:
Yes as of now, we can change the constraint, and running both would take a lot of time

Reply:
Cost wise 12x is 3 times more expensive than 4x. I'm wondering if we want to default to 4x unless the benchmarking time is significantly faster considering the overheads for instance startups.

Reply:
We run this once a week, so it should not be too much.
If we end up using this too much I can always change the constraint

Reply:
Another aspect to consider is that what our customers care most? Do they mostly run on single GPU or multi-GPUs? Do we even want to run on both on weekly basis?@tonyhoo @zhiqiangdon might chime in if you have a chance.

Reply:
For now only the maintainers can run this to prevent misuse of resources.

Reply:
Are tabular and time series using the default instance type (g4dn.2x) for benchmark with your current setup?

Reply:
no they're using m5.2x

Reply:
For tabular, I believe we want to use `m6i.2xlarge`, but I'm not sure about time series.